### PR TITLE
Update installing-istio.md based on new archive format

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: _docs/tasks/installing-istio.md [hunk 1]
```diff
@@ -79,13 +79,10 @@ default namespace. They can be modified for deployment in a different namespace.
    source ../../istio.VERSION
    ```
 
-6. Install istioctl CLI. Use one of the [`istioctl`]({{home}}/docs/reference/commands/istioctl.html) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
-`istioctl-linux`, targeted at Mac, Windows or Linux users respectively. For example, run the following commands on a Mac system:
+6. Install istioctl CLI. Use one of the [`istioctl`]({{home}}/docs/reference/commands/istioctl.html) client binaries corresponding to your OS: `istioctct/osx/istioctl`, `istioctct/windows/istioctl.exe`, `istioctl/linux/istioctl`, targeted at Mac, Windows or Linux users respectively. For example, run the following commands on a Mac system:
```
Comment (ldemailly) [inline _docs/tasks/installing-istio.md]: typos... istioctct instead of ctl

Comment (sebastienvas) [inline _docs/tasks/installing-istio.md]: Done

File: _docs/tasks/installing-istio.md [hunk 2]
```diff
@@ -31,12 +31,12 @@ This page shows how to install and configure Istio in a Kubernetes cluster.
 For the {{ site.data.istio.version }} release, Istio must be installed in the same Kubernetes namespace as the applications. Instructions below will deploy Istio in the
 default namespace. They can be modified for deployment in a different namespace.
 
-1. Go to [istio release page](https://github.com/istio/istio/releases), and download and extract the installation files istioctl.tar.gz and the source code.
+1. Go to [istio release page](https://github.com/istio/istio/releases), and download and extract the installation files istio.tar.gz.
```
Comment (geeknoid) [inline _docs/tasks/installing-istio.md]: Put istio.tar.gz inside of backticks

Comment (sebastienvas) [inline _docs/tasks/installing-istio.md]: done

**Pull request merged**